docs/sidebars.js

```diff
@@ -6,49 +6,145 @@ const path = require("path");
 /** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
 export default {
   sidebar: [
+   
```

Comment:
Not sure where the top bar is, so adding comment here. Protocol specs is now tucked away in the middle of the long/mixed "Resources" dropdown menu.
If not it's own item, I feel it would be more fitting in the "References" section.

Reply:
i think it should be tucked away. its like a whitepaper

Reply:
This protocol specification is important for early engineers building on Aztec, particularly devnets towards testnet. Doesn't need to hold the PR though.
![image](https://github.com/user-attachments/assets/97b4502c-04c4-4350-ab3f-8339d27fedc9)